gr-blocks/lib/rotator_cc_impl.cc

```diff
@@ -52,6 +60,19 @@ void rotator_cc_impl::set_phase_inc(double phase_inc)
     d_r.set_phase_incr(exp(gr_complex(0, phase_inc)));
 }
 
+void rotator_cc_impl::handle_phase_inc_msg(pmt::pmt_t msg)
+{
+    if (pmt::is_pair(msg)) {
```

Comment:
What if not? Please log a warning or other debug message.

Reply:
Do you prefer a print/warning message or really a `throw std::runtime_error`? I've put the latter, but let me know what you think.

Reply:
I would have just logged the error and skipped the message, but I have no objective reason to say so.